tests/unit/test_g2p.py

```diff
@@ -0,0 +1,208 @@
+"""
+Unit tests for genotypephenotype objects. This is used for all tests
+that can be performed in isolation from input data.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import logging
+
+import ga4gh.frontend as frontend
+import ga4gh.protocol as protocol
+
+
+class TestGenotypePhenotypeSearch(unittest.TestCase):
+    """
+    Tests the basic routing and HTTP handling for
```

Comment:
We don't really care about the HTTP level stuff here  as this is handled by other tests. What we really want to see are some unit tests of the classes in the G2P module. I wouldn't bother with the flask routing thing here, and instead try instantiating your backend directly. You could run some tests using the client API based on the data that you have in the tests directory.